Not needed to run tests and we do not have them packaged

--- urllib3-2.1.0/dev-requirements.txt.orig
+++ urllib3-2.1.0/dev-requirements.txt
@@ -8,5 +8,3 @@
 trustme==1.1.0
 cryptography==41.0.4
 backports.zoneinfo==0.2.1;python_version<"3.9"
-towncrier==23.6.0
-pytest-memray==1.5.0;sys_platform!="win32" and implementation_name=="cpython"
